organisaatio-ui/src/components/Taulukot/Normaalitaulukko.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
good starting point for adding more test... 
- feed in some props, see it they are transformed properly
- what happens on input events?
- ...so forth